appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/node/LocalStrings.properties

```diff
@@ -42,5 +42,6 @@ enterprise.deployment.node.invaliddeploymentdescriptors=Invalid Deployment Descr
 
 enterprise.deployment.node.invalidvalue=Invalid value for a tag under {0} : {1}
 invalid.root.element={0} Element [{1}] is not a valid root element
+invalid.schema=Requested schema not in local repository, check for typos in the descriptor file
```

Comment:
Please rephrase the above message as

"Requested schema is not found in local repository, please ensure that there are no typos in the XML namespace declaration."